source/configuration_manual/mail_location/obox/http_storages.rst

```diff
@@ -82,7 +82,14 @@ The parameters common to all object storages include:
 +---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------+--------------+
 | max_connect_retries=<n>               |Number of connect retries                                                                                                      | 2            |
 +---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------+--------------+
-| max_retries=<n>                       |Max number of HTTP request retries                                                                                             | 4            |
+| max_retries=<n>                       |Max number of HTTP request retries. Retries happen for 5xx errors as well as for 423(locked)                                   | 4            |
+|                                       |with :ref:`sproxyd <scality_sproxyd>` and 409(conflict) with :ref:`cdmi <scality_cdmi>`.                                       |              |
+|                                       |There is a wait between attempting next retry. The initial retry is done after 50ms. The following retries are done            |              |
+|                                       |after waiting ten times as long as the previous attempt, so 50ms -> 500 ms -> 5s ->10s. The maximum wait time before retry is  |              |
```

Comment:
I think this refers to the absolute/cumulative total of 10 seconds? Could it be clarified with wording like "The maximum cumulative wait.."

Reply:
No, in this case here it's a 10 second retry interval. There's absolute_timeout for the cumulative timeout. Wouldn't hurt to mention that of course.

Reply:
The default(`max_retries=4`) cumulative wait is a always more then 15.55 seconds. If people configure more or less max_retries that can change.  I added "per attempt" to make it clearer.

If the request times out a warning is generated. If it times out depends on what is configured for `timeout`. I added a sentence about that as well. It's slightly longish now but i think that helps getting the whole picture.

Reply:
Oh i did not see Timo's earlier comment. I will try update it with absolute_timeout.

Reply:
Does it make sense now?